### PR TITLE
docs: Add information about available packages

### LINKED ISSUE #1: AUR & Nix Packages
Hi Jon,

First off I want to thank you for creating this utility, it has made Hyprland usable for me :)

I was building the package locally in order to get the fixes for Hyprland 0.40 and figured I may as well create a package on the AUR. The link to the package is https://aur.archlinux.org/packages/yambar-hyprland-wses.

I also noticed in my searching that someone has created a nix package although this is using v0.1.0 (https://github.com/NixOS/nixpkgs/blob/nixos-23.11/pkgs/applications/misc/yambar-hyprland-wses/default.nix)

I am creating this issue to see if you are interested in a pull request to add these packages to the documentation.

### PR DESCRIPTION
**Pull request by CarwynNelson**
There are community maintained packages for Arch and Nix which some may prefer to use. Highlight these in the documentation.

Resolves #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,46 @@
...
+
 Assuming you have [Rust installed][Rust], run:
 
 ```console
 $ cargo install yambar-hyprland-wses
 ```
 
-and then use the Yambar config further down in the README. You'll need
-to adjust the `script` path.
+Use `/home/yourusernamehere/.cargo/bin/yambar-hyprland-wses` as your script path
```
Comment (jonhoo) [inline README.md]:
```suggestion
Use `~/.cargo/bin/yambar-hyprland-wses` as your script path
```

File: README.md [hunk 2]
```diff
@@ -1,13 +1,46 @@
...
+
+## Arch Linux
+
+There is a community maintained package available in the AUR. Assuming you have
+[yay], run:
```
Comment (jonhoo) [inline README.md]: We could also link to [paru](https://github.com/Morganamilo/paru) which is also written in Rust (like this is!). It's what I personally use. Maybe insert a generic mention here of "any AUR helper". Could also give the generic AUR install instructions:

```console
$ git clone https://aur.archlinux.org/yambar-hyprland-wses.git
$ cd yambar-hyprland-wses
$ makepkg -si
```

File: README.md [hunk 3]
```diff
@@ -1,13 +1,46 @@
...
+```nix
+environment.systemPackages = [
+  pkgs.yambar-hyprland-wses
+];
+```
+
+Yambar does not check the `$PATH` for scripts which means you cannot just use
+`yambar-hyprland-wses` as the script path. It is probably easiest for Nix users
+to generate the yambar config in your nix config, which will know the correct
+path to `yambar-hyprland-wses`.
```
Comment (jonhoo) [inline README.md]: Maybe include a line for how they get that path if they nixify their config?

File: README.md (hunk 1 above)
Comment (jonhoo) [reply README.md]: I recently discovered that yambar specifically supports paths that start with `~/` (ref yambar-modules-script(5))

**Review by jonhoo — CHANGES_REQUESTED** [review]
Nice! Mostly nits :)

**Comment by CarwynNelson** [comment]
> Nice! Mostly nits :)

Nits welcome :)

File: README.md (hunk 1 above)
Comment (CarwynNelson) [reply README.md]: Nice to know that exists; I have updated the docs to reflect that.

File: README.md (hunk 2 above)
Comment (CarwynNelson) [reply README.md]: Ah yes it does feel right for a Rust utility to mention a Rust AUR helper :)

I have added instructions for Paru and manual installation

File: README.md (hunk 3 above)
Comment (CarwynNelson) [reply README.md]: I did a bit of digging and found out how you generate that path in nix config.

I must admit I have barely used Nix so I'm no expert on it. I'm wondering if its worth either:
1. Removing the section on the Nix package as it is mostly a guess, or
2. Simplifying the section to note that there is a nix package if people want it, but you are on your own a little bit with configuration, and PRs for better docs on the Nix package are welcome

Do you have any thoughts?

**Comment by CarwynNelson** [comment]
I have updated the PR with the requested changes. Thanks for taking the time to review my pr I really appreciate it :)

File: README.md (hunk 3 above)
Comment (jonhoo) [reply README.md]: Let's go with option 2 :+1:

File: README.md [hunk 4]
```diff
@@ -13,7 +13,7 @@ Assuming you have [Rust installed][Rust], run:
 $ cargo install yambar-hyprland-wses
 ```
 
-Use `/home/yourusernamehere/.cargo/bin/yambar-hyprland-wses` as your script path
+Use `/.cargo/bin/yambar-hyprland-wses` as your script path
```
Comment (jonhoo) [inline README.md]: Ah, this isn't quite right. You need the `~` at the beginning of the path to indicate the home directory, and the same in the YAML down below.

Comment (CarwynNelson) [reply README.md]: Wow, I swear I added `~` at the front :sweat_smile:

I'll get that sorted.

**Review by jonhoo — APPROVED** [review]
Looks great, thank you!

**Pull request merged**